client/lobby/OptionsTab.h

```diff
@@ -105,8 +106,14 @@ class OptionsTab : public OptionsTabBase
 		const int TEXT_POS_X = 29;
 		const int TEXT_POS_Y = 56;
 
+		const int MAX_LINES = 6;
+		const int MAX_ELEM_PER_LINES = 4;
+
 		int elementsPerLine;
 
+		std::shared_ptr<CSlider> slider;
+		int sliderLine;
```

Comment:
Probably needs better name?
Also - why can't you acquire this information from slider?

If this is because slider may not exists then add method that would either return value from slider or 0 if there is no slider and remove this member entirely